### PR TITLE
Updated a readme instruction step and synced up all the readmes

### PR DESCRIPTION
**Pull request by shpark76**

### CODE REVIEW

File: wherehows-backend/README.md [hunk 1]
```diff
@@ -66,23 +62,18 @@ URL	 jdbc:mysql://<metastore host>:3306/metastore
...
- lib/mysql-connector-java-5.1.36.jar
-```
 
 ### Run
 To run the backend service:
 
 Set the variables in application.env to configure the application.
 
-To Run backend service application on port 19001 (from the wherehows-backend folder):
+To Run backend service application on port 9001 (from the wherehows-backend folder):
```
Comment (shpark76) [inline wherehows-backend/README.md]: if it's an idea, we should change other configuration(ie. Docker) and readme to make consistency.

Comment (mars-lan) [inline wherehows-backend/README.md]: By default it's running on 19001: https://github.com/linkedin/WhereHows/blob/master/wherehows-backend/build.gradle#L4

Comment (shpark76) [inline wherehows-backend/README.md]: Feel free to add your idea on this, I will update all related docs and configs. 
- Backend: 19001
- Frontend: 9000
- Elasticsearch: 9200
- MySQL: 3306

Comment (mars-lan) [inline wherehows-backend/README.md]: Actually let's use 9001 for front end. Thanks.

**Pull request merged**